### DISCUSSION TITLE
Webhook mapping

### DISCUSSION THREAD

**Discussion by oivarc** [pimcore/pimcore#17942] [Data-Hub]
Hello,

I am managing a Pimcore installation for a client with the enterprise license. We tried to configure a webhook on the postDelete event for an object, we used in the GridMapping the ID (system property) and a DateTime field we have on the object.

When the event is triggered the webhook processor generate an error saying that the setId() on the object needs an 'int' but a string is passed and that the setDate() needs a 'Carbon' but a string is passed.

Is this a bug or there is another way to map that kind of fields ?

**Comment by fashxp** [comment]
Hmm, sounds more like a bug to me. 
Please report that issue via our enterprise portal, thx.
